lc-hax/Scripts/Commands/GrabCommand.cs

```diff
@@ -17,7 +17,7 @@ public class GrabCommand : ICommand {
         Vector3 positionOffset,
         Transform parentObject
     ) {
-        HaxObjects.Instance?.GrabbableObjects?.WhereIsNotNull().ForEach(grabbableObject => {
+        GrabbableDependencyPatch.ActiveProps.WhereIsNotNull().ForEach(grabbableObject => {
```

Comment:
Can we convert this to a helper as well to be consistent with `Helper.Enemies`? So a `Helper.GrabbableObjects`?

Reply:
Done